hoomd/operation.py

```diff
@@ -547,6 +547,9 @@ class see its documentation.
         + inspect.cleandoc(AutotunedObject._doc_inherited)
     )
 
+    def __init__(self):
+        self._allow_if_updater = False
+
```

Comment:
```suggestion
```
I don't think this is necessary. See my comment where `_allow_if_updater` is used.